### PR TITLE
Updates-style-guide

### PR DESCRIPTION
**Pull request by natebower**
Signed-off-by: natebower <102320899+natebower@users.noreply.github.com>

### Description
_Updates style guide_

### Checklist
- [ ] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by natebower** [comment]
@kolchfa-aws Can you give me a sanity check and let me know if you think there's anything else I should add?

File: STYLE_GUIDE.md [hunk 1]
```diff
@@ -56,13 +56,87 @@ Avoid excessive words, such as please. Be courteous but not wordy. Extra detail
...
+
+In general, spell out acronyms once on a page. However, you can spell them out more often for clarity.
+
+Make an acronym plural by adding an *s* to the end of it. Do not add an apostrophe.
+
+How an acronym is pronounced determines whether you use the article *an* or *a* before it. If it's pronounced with an initial vowel sound, use *an*. Otherwise, use *a*.
+
+If the first use of an acronym is in a heading, retain the acronym in the heading, and then write out the term in the following body text, followed by the acronym in parentheses. Don't spell out the term in the heading with the acronym included in parentheses. If the first use of the service name is in a title or heading, use the short form of the name in the heading, and then use the long form followed by the short form in parentheses in the following body text.
+
+In general, spell out abbreviations that end with *-bit* or *-byte*. Use abbreviations only with numbers in specific measurements. Always include a space between the number and unit. Abbreviations that are well known and don't need to be spelled out on are *KB*, *MB*, *GB*, and *TB*.
```
Comment (vagimeli) [inline STYLE_GUIDE.md:73]: Delete "on" in the last sentence.

File: TERMS.md [hunk 2]
```diff
@@ -608,6 +618,10 @@ Avoid using as a verb to refer to an action that precipitates a subsequent actio
 
 A storage tier that you can use to store and analyze your data with Elasticsearch and Kibana that is optimized for performance. To learn more about the service, see the introductory [blog post](https://aws.amazon.com/about-aws/whats-new/2020/05/aws-announces-amazon-elasticsearch-service-ultrawarm-general-availability/).
 
+**upper left, upper right**
```
Comment (Naarcha-AWS) [inline TERMS.md:621]: If we usually use upper left or upper right as adjectives, should we hyphenate upper right and upper left on this line?

**Review by Naarcha-AWS — APPROVED** [review]
One small question. Otherwise, LGTM

**Review by vagimeli — APPROVED** [review]
Minimal comments. LGTM.

File: STYLE_GUIDE.md [hunk 3]
```diff
@@ -56,13 +56,87 @@ Avoid excessive words, such as please. Be courteous but not wordy. Extra detail
...
+          - If you are linking to procedures, you can use either "For instructions *on*" or "instructions *for*." Don't use "instructions *about*."
+          - Where space is limited (for example, in a table), you can use "*See* [link text]." Don't use *go to*.
+     - Ensure that the link text matches the section title text. <br> <br> Example: "To get involved, see [Contributing](https://opensearch.org/source.html) on the OpenSearch website." <br> <br>
+
+- **Embedded links**: Embedded links are woven into a sentence without formal introductory text. They're especially useful in tables or other elements where space is tight. The text around the embedded link must relate to the information in the link so that the reader understands the context. Do not use *here* or *click here* for link text because it creates accessibility problems. <br> <br> Example: "Finally, [delete the index](https://opensearch.org/docs/latest/api-reference/index-apis/delete-index)."
+
 ### Punctuation and capitalization
 
+- Use sentence case for titles and headings. Titles of standalone documents may use title case.
+
```
Comment (kolchfa-aws) [inline STYLE_GUIDE.md:134]: Do we want to mention that column headers should be sentence case here?

**Review by kolchfa-aws — APPROVED** [review]
LGTM.

**Pull request merged**